src/hooks/util/parseMarkdown.tsx

```
@@ -87,11 +87,14 @@ function parseMarkdown(text: string): string {
+  content,
+  className,
+}: {
+  content: string;
+  className: string;
+}) {
   return (
-    <div
-      className="text-sm leading-relaxed"
-      dangerouslySetInnerHTML={{ __html: parseMarkdown(content ?? '') }}
-    />
+    <div className={className} dangerouslySetInnerHTML={{ __html: parseMarkdown(content ?? '') }} />
```

Biome (2.4.6)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)